python/sglang/srt/layers/fused_moe_triton/fused_moe_splitk.py

```diff
@@ -0,0 +1,867 @@
+# Adapted from https://github.com/pytorch-labs/applied-ai/blob/main/kernels/triton/inference/col_major_moe_gemm/v1_moe_fused.py
```

Comment:
Initially, I didn't consider putting the splitk implementation in a separate file. However, keeping it in the same file as `fused_moe.py` would require additional conditional logic, and importing common functions from `fused_moe.py` might lead to circular import issues. After consideration, I decided to create a new file.